pkg/grpc/proxy/codec/codec.go

```diff
@@ -22,7 +22,7 @@ func init() {
 
 // Register manually registers the codec.
 func Register() {
-	encoding.RegisterCodec(codec())
+	// encoding.RegisterCodec(codec())
```

Comment:
Is this needed or not? If it's needed, why? (I recall mention of a gRPC issue but wasn't sure what the resolution was)

Reply:
It does not appear to be needed any longer. It was most likely an error on my part